### PR TITLE
Stable/5.0 - Auto 104 - Snapshot - Automate snapshot-cancel CLI test

### PR DESCRIPTION
**Pull request by priyankagupte15**
image

### CODE REVIEW

File: tempest/api/workloadmgr/snapshot/test_snapshots.py [hunk 1]
```diff
@@ -215,10 +215,110 @@ def test_3_list_snapshot(self):
...
+                    tvaultconf.PASS)
+                LOG.debug("Command executed correctly")
+
+            time.sleep(5)
+            snapshot_id = query_data.get_inprogress_snapshot_id(workload_id)
+            LOG.debug("Snapshot ID: " + str(snapshot_id))
+
+            # Snapshot cancel CLI with invalid options
+            snapshot_cancel_novalue = command_argument_string.snapshot_cancel
+            err_msg1 = "workloadmgr snapshot-cancel: error: the following arguments are required: <snapshot_id>"
```
Comment (nasharatm) [inline tempest/api/workloadmgr/snapshot/test_snapshots.py]: Can we move this error_msg string to tvaultconf.py?

File: tempest/api/workloadmgr/snapshot/test_snapshots.py [hunk 2]
```diff
@@ -215,10 +215,110 @@ def test_3_list_snapshot(self):
...
+                reporting.add_test_step("Snapshot cancel cli with no option returned correct error",
+                                        tvaultconf.PASS)
+            else:
+                LOG.debug("Snapshot cancel cli with no snapshot id returned no error")
+                reporting.add_test_step("Snapshot cancel cli with no snapshot id returned correct error",
+                                        tvaultconf.FAIL)
+
+            # Snapshot cancel CLI with invalid options
+            snapshot_cancel_invalid = command_argument_string.snapshot_cancel + "invalid"
+            err_msg2 = "ERROR:workloadmgr:No snapshot with a name or ID of 'invalid' exists."
```
Comment (nasharatm) [inline tempest/api/workloadmgr/snapshot/test_snapshots.py]: Can we move this error_msg string to tvaultconf.py?

**Review by nasharatm — CHANGES_REQUESTED** [review]

File: tempest/api/workloadmgr/snapshot/test_snapshots.py (hunk 2 above)
Comment (priyankagupte15) [reply tempest/api/workloadmgr/snapshot/test_snapshots.py]: done

File: tempest/api/workloadmgr/snapshot/test_snapshots.py (hunk 1 above)
Comment (priyankagupte15) [reply tempest/api/workloadmgr/snapshot/test_snapshots.py]: done

**Review by nasharatm — APPROVED** [review]

**Review by SuhasTA4 — APPROVED** [review]

**Pull request merged**